### PR TITLE
deprecate Tink worker global timeout cli flag:

### PR DESCRIPTION
**Pull request by jacobweinstock**
## Description

The Tink worker global timeout cli flag doesn't implement the global timeout defined in a Tink server template correctly.

A single template can have multiple workers. So a single Tink worker defining a global timeout for all tasks is erroneous. It doesn't take the multiple workers idea into account.

A long term solution will need to be implemented. Something that takes into account multiple workers. This PR just removes this erroneous behavior.

## Why is this needed

Fixes: #

## Checklist:

I have:

- [ ] updated the documentation and/or roadmap (if required)
- [ ] added unit or e2e tests
- [ ] provided instructions on how to upgrade

### CODE REVIEW

File: cmd/tink-worker/cmd/root.go [hunk 1]
```diff
@@ -59,9 +59,7 @@ func NewRootCommand(version string, logger log.Logger) *cobra.Command {
 
 			ctx := context.Background()
 			if timeOut > 0 {
-				var cancel context.CancelFunc
-				ctx, cancel = context.WithTimeout(ctx, timeOut)
-				defer cancel()
+				logger.Info("timeOut is deprecated")
```
Comment (mmlb) [inline cmd/tink-worker/cmd/root.go:62]: lets just get rid of the code, we have enough workarounds/deprecated things and it just turns into baggage. We're still at v <1.0 and also have sandbox to provide a cushion for breakages.

**Review by mmlb — CHANGES_REQUESTED** [review]

File: cmd/tink-worker/cmd/root.go (hunk 1 above)
Comment (jacobweinstock) [reply cmd/tink-worker/cmd/root.go:62]: sure thing

File: cmd/tink-worker/cmd/root.go [hunk 2]
```diff
@@ -96,6 +86,7 @@ func NewRootCommand(version string, logger log.Logger) *cobra.Command {
 				worker.WithLogCapture(captureActionLogs),
 				worker.WithPrivileged(true))
 
+			ctx := context.Background()
 			err = w.ProcessWorkflowActions(ctx)
```
Comment (micahhausler) [inline cmd/tink-worker/cmd/root.go:90]: Can you use the command context here?
```suggestion
			err = w.ProcessWorkflowActions(cmd.Context())
```

**Review by micahhausler — CHANGES_REQUESTED** [review]
Thanks for doing this! Just one minor change

**Review by micahhausler — APPROVED** [review]
LGTM

File: cmd/tink-worker/cmd/root.go (hunk 2 above)
Comment (mmlb) [reply cmd/tink-worker/cmd/root.go]: This is currently effectively the same thing. Will change if we do `ExecuteContext` in cmd/tink-worker/main.go which I think would give us ctrl-c support? For a later PR though.

**Review by mmlb — APPROVED** [review]

**Pull request merged**